Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.6
+
+ADD . /app
+
+RUN cd app && sh docker/container-install.sh
+
+WORKDIR /app
+
+ENV MYSQL_HOST localhost
```

Comment:
My thinking is that for testing purposes it will have a db sitting inside the image. This means you can just run the container and cytube works. That said, you are right, it is always more elegant to have the image only contain the software and nothing else, and if you need a db you make another service (container or otherwise) and connect to it.